src/Downloader.php

```diff
@@ -180,14 +180,20 @@ protected function fetchCertificates(string $hostName): array
             $connectTo = $hostName;
         }
 
-        $client = @stream_socket_client(
-            "ssl://{$connectTo}:{$this->port}",
-            $errorNumber,
-            $errorDescription,
-            $this->timeout,
-            STREAM_CLIENT_CONNECT,
-            $streamContext
-        );
+        try {
+            $client = stream_socket_client(
+                "ssl://{$connectTo}:{$this->port}",
+                $errorNumber,
+                $errorDescription,
+                $this->timeout,
+                STREAM_CLIENT_CONNECT,
+                $streamContext
+            );
+        }
+        catch (\ErrorException $ex) {
+            $client = null;
```

Comment:
I don't think setting the client to null is necessary. Could you remove this?

Reply:
`$client` is checked a few lines later, because the variable always used to exist, but not anymore:

    if (! $client) {